roles/windows-executor/defaults/main/install_dev_tools_defaults.yml

```diff
@@ -1,14 +1,14 @@
-nunit_console_runner_version: 3.16.0
-nano_version: 7.0.7
-vim_version: 9.0.0975
+nunit_console_runner_version: 3.16.2
+nano_version: 7.2.14
+vim_version: 9.0.1221
 jq_version: 1.6
-golang_version: 1.19.3
-openjdk_version: 19.0.1
+golang_version: 1.20.1
+openjdk_version: 19.0.2
 miniconda3_version: 4.12.0
 python_3_version: 3.11.0
-nodejs_version: 19.2.0
+nodejs_version: 19.6.0
```

Comment:
19.6.1 is the latest current

Reply:
Done - thanks

Reply:
The build ended up failing. I was wondering why my version was older, it's because the .1 release isn't available in Chocolatey yet.

Reply:
@felicianotech Seems like it is, but under a slightly different package (not sure what the difference is tbh) https://community.chocolatey.org/packages/nodejs.install

Perhaps we should consider moving to NVM in the future?